packages/components/message-box/src/index.vue

```diff
@@ -270,6 +281,8 @@ export default defineComponent({
       () => state.icon || TypeComponentsMap[state.type] || ''
     )
     const hasMessage = computed(() => !!state.message)
+    const rootRef = ref<ComponentPublicInstance>(null)
+    const headerRef = ref<ComponentPublicInstance>(null)
```

Comment:
**Question:**  I think this one should be `HTMLElement` per your usage on the template, right?